src/app/components/homepage/cards-container/card-content-detail/card-content-detail.component.ts

```diff
@@ -21,6 +21,9 @@ export class CardContentDetailComponent implements OnInit, OnDestroy {
     type: string;
     counterLike: number;
     counterDisl: number;
+    private userId: string;
+    name: string;
+    ava: string;
     private userSub: Subscription;
     private routeSubscription: Subscription;
```

Comment:
this component has destroy$ already, supposes you can avoid additional subscriptions